src/channels/CreateDidChannels/types.ts

```diff
@@ -0,0 +1,12 @@
+import { HexString } from '@polkadot/util/types';
+
+import { DAppName } from '../AccessChannels/DAppName';
+import { Origin } from '../AccessChannels/Origin';
+
+export type CreateDidInput = DAppName;
+
+export type CreateDidOriginInput = DAppName & Origin;
+
+export interface CreateDidOutput {
+  signed: HexString;
```

Comment:
Is this what is given to the dApp?
If yes, than I would be more explicit. Like `signedExtrinsic`